app/views/feedback/steps/_signup.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Give feedback on this sign-up form </h1>
+
+<p>
+  Tell us about your experience signing up for <%= feedback_session['action'].split.last %>
```

Comment:
Can this be updated to display 'an adviser' instead of 'adviser', and same for 'an event'? And a full stop at the end of the sentence?